### PR TITLE
Colorize toolbar background

### PR DESCRIPTION
**Pull request by ricardoboss**
From this:

image

To this:

image

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -84,6 +84,7 @@
 
 .acrylic {
 	backdrop-filter: blur(5px);
+	background: rgba(var(--v-theme-primary), 0.75) !important;
```
Comment (chucker) [inline src/App.vue:87]: maybe 0.67

**Review by chucker — APPROVED** [review]

File: src/App.vue (hunk 1 above)
Comment (ricardoboss) [reply src/App.vue:87]: image

ok?

Comment (ricardoboss) [reply src/App.vue:87]: image

0.5 might be too transparent

Comment (chucker) [reply src/App.vue]: Yeah, 0.66 looks good.

**Pull request merged**